acnportal/acnsim/network/charging_network.py

```diff
@@ -413,16 +421,23 @@ def constraint_current(
         Returns:
             np.ndarray: Aggregate currents subject to the given constraints.
         """
+        # If there are no constraints, throw an error as this function is not well-
+        # defined.
+        # TODO: Test the no constraint case for is_feasible and constraint_currents.
+        if self.constraint_matrix is None:
```

Comment:
@zach401 Does this look good for the None case? If so I'll write a test

Reply:
I think we should try to support the no constraint case. I thought this was already covered, but maybe not?

Reply:
is_feasible returns True if called when no constraints are added. It's constraint_current that throws an error if called without constraints since there aren't any currents to constrain. But we could also just return an empty NumPy array. I think that's our best bet if we want to avoid throwing an error. I think not erroring, in this case, is fine as long as we're clear about the behavior in the docstring. @zach401 sound good?

Reply:
To clarify, I'd return `np.array([])` if `self.constraint_matrix` is None (and add a test).